build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.2.0-alpha16"
+        classpath 'com.android.tools.build:gradle:7.0.0-alpha04'
```

Comment:
Could you leave the quotes as `"`? `"` is useful for string templates, and for consistency it's better to use them everywhere